jsconcat.php

```diff
@@ -31,8 +31,6 @@ function __construct( $scripts ) {
 			}
 			unset( $this->$key );
 		}
-
-		parent::__construct();
```

Comment:
The parent constructor is called here? https://github.com/Automattic/nginx-http-concat/blob/master/jsconcat.php#L19-L23

Either we pass in a WP_Scripts object (so it has already been called) or we instantiate a new WP_Scripts object, which is the same as calling the constructor.